collect_app/src/main/java/org/odk/collect/android/openrosa/OpenRosaResponseParserImpl.kt

```diff
@@ -0,0 +1,220 @@
+package org.odk.collect.android.openrosa
```

Comment:
What do you think about moving it to its own module?

Reply:
Definitely! I've looked at this a few times before, but I'm not sure if we've got all the ties to `collect_app` severed yet. I think I'd rather leave that for another time rather than adding to this PR.

Reply:
Yes yes not in this pr.